ccip/ccip.Dockerfile

```
@@ -0,0 +1,6 @@
+FROM chainlink-ccip:latest
```

Is there a way to write this parameterized so that we can reference official chainlink release images? Maybe like:
```suggestion
ARG TAG="latest"
FROM chainlink:${TAG}
```